compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeHostingViewController.uikit.kt

```diff
@@ -88,14 +88,29 @@ internal class ComposeHostingViewController(
     private val lifecycleOwner = ViewControllerBasedLifecycleOwner()
     private val hapticFeedback = CupertinoHapticFeedback()
 
+    private val metalView = MetalView(
```

Comment:
It is not clear what view it is. for the initial mediator? for Popups?

Reply:
Renamed to `rootMetalView`